pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -145,7 +145,11 @@ def synchronize(remote_pk, repository_pk, mirror, skip_types):
                                            repository=sub_repo)
                 dv.create()
 
-    first_stage = RpmFirstStage(remote, deferred_download, skip_types, treeinfo=treeinfo)
+    first_stage = RpmFirstStage(remote,
+                                repository,
```

Comment:
for kickstarts, the main repo is the last one to be synced, so for speed up, I think we need to pass `repository``to the [subrepos](https://github.com/pulp/pulp_rpm/pull/1606/files#diff-7a569a289e63f956d6451ebe97f33a96R143) also.